### PR TITLE
Updating documentation 

### PR DESCRIPTION
**Pull request by asbanger**
## Description
This PR updates the documentation with the following changes:
- Updating the GitHub codespaces documentation
- Fixing minor typos and errors

## Motivation and Context
The purpose of this PR is to improve the documentation for better readability and clarity.

## How Has This Been Tested?
This PR has been tested by reviewing the changes made and ensuring that the documentation is accurate and up-to-date.

## Types of changes
- [x] Docs update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: docs/deployment/deployment.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ Begin by first forking the Information Assistant repository into your own reposi
...
 ![Fork the Information Assistant Repo](/docs/images/fork_repo.png)
 
-Once you have forked the repo, you can then use the following button to open the Information Assistant Codespace. You will need to select your forked repo and the location for your Codespace to run in.
+Once you have forked the repo, you can then use the following button to open the Information Assistant GitHub Codespaces. You will need to select your forked repo and the location for your GitHub Codespaces to run in.
 
-[![Open in GitHub Codespaces](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=false&ref=main&machine=basicLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json)
+[![Open in GitHub Codespacess](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=false&ref=main&machine=basicLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json)
```
Comment (dayland) [inline docs/deployment/deployment.md:18]: Too many "S"s

File: docs/deployment/developing_in_a_GitHub_Codespaces.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+  - [Getting started with GitHub Codespaces](#getting-started-with-GitHub-Codespaces)
+  - [Creating your GitHub Codespaces](#creating-your-codespace)
   - [Using GitHub Codespaces in Visual Studio Code](#using-github-codespaces-in-visual-studio-code)
   - [Prerequisites](#prerequisites)
-  - [Opening a codespace in VS Code](#opening-a-codespace-in-vs-code)
+  - [Opening a GitHub Codespaces in VS Code](#opening-a-codespace-in-vs-code)
   - [Navigating to an existing codespace](#navigating-to-an-existing-codespace)
 ---
-## Getting started with codespace
+## Getting started with GitHub codespaces
```
Comment (dayland) [inline docs/deployment/developing_in_a_GitHub_Codespaces.md:11]: We need to be consistent throughout. In some cases we have "GitHub Codespaces" and in other "GitHub codespaces". The capitalization needs to be consistent.

File: docs/deployment/developing_in_a_GitHub_Codespaces.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
+## Creating your GitHub codespaces
 
-1. Click on    [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/)
-1. New window will open where you can create a new Codespace
+1. Click on    [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new/)
+1. New window will open where you can create a new codespace
 1. First you will need to select your repository you forked from the Information Assistant repo
-1. Next, by default you will be on **main** branch, you can switch to specific branch where you would like to create a Codespace.
-1. Then select options for **Region** and **Machine type**. The "Dev Container configuration" will be prepopulated and does not need to be changed.
+1. Next, by default you will be on **main** branch, you can switch to specific branch where you would like to create a Codespaces.
```
Comment (dayland) [inline docs/deployment/developing_in_a_GitHub_Codespaces.md:21]: when "codespace" is used not in conjunction with "GitHub Codespaces" it should not be capitalized.

**Review by dayland — APPROVED** [review]

File: docs/deployment/developing_in_a_GitHub_Codespaces.md (hunk 2 above)
Comment (asbanger) [reply docs/deployment/developing_in_a_GitHub_Codespaces.md:11]: Fixed

File: docs/deployment/developing_in_a_GitHub_Codespaces.md (hunk 3 above)
Comment (asbanger) [reply docs/deployment/developing_in_a_GitHub_Codespaces.md:21]: Sure. I will update accordingly

**Review by ryonsteele — DISMISSED** [review]

File: docs/deployment/deployment.md (hunk 1 above)
Comment (dayland) [reply docs/deployment/deployment.md:18]: I still see two "S" in the link.. 
image

**Review by dayland — APPROVED** [review]

**Review by ryonsteele — APPROVED** [review]

**Pull request merged**